scripts/snaptron/process_tp53_junctions.py

```diff
@@ -38,15 +38,15 @@ def get_rail_id(row):
     return(all_sample_ids)
 
 # File paths
-sample_file = 'samples.tsv'
-junction_file = 'tp53_junctions.txt'
+sample_file = 'samples.tsv.gz'
+junction_file = 'tp53_junctions.txt.gz'
 mut_scores_file = os.path.join('..', '..', 'classifiers', 'TP53',
                                'tables', 'mutation_classification_scores.tsv')
 out_junc_file = 'junctions_with_mutations.csv'
 
-samples = pd.read_table(sample_file, low_memory=False)
+samples = pd.read_table(sample_file, compression='gzip', low_memory=False)
 dictionary = samples[['rail_id', 'gdc_cases.samples.submitter_id']]
-junc = pd.read_table(junction_file)
+junc = pd.read_table(junction_file, compression='gzip')
```

Comment:
are these zipped because they are (relatively) large files? perhaps also compress outputs then? pandas 17+ works with to_csv(compression='gzip')

Reply:
that's a great idea. I will do and then merge. Thanks!